build/sbt-config/repositories-cn.template

```diff
@@ -0,0 +1,7 @@
+[repositories]
```

Comment:
does the file support comments? we can add comments to guide user how to use this file

Reply:
https://stackoverflow.com/questions/56343343/how-do-you-comment-in-the-sbt-repositories-file

The commenting style commences with the symbol `#`